### PR TITLE
SpeedDreamsTeam.SpeedDreams version 2.3.0

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106964)

### CODE REVIEW

**Comment by stephengillie** [comment]
This package is a 1.77 GB download.

**Comment by stephengillie** [comment]
This package installed normally, but errored when launched from the start menu shortcut:
> Faulting application name: speed-dreams-2.exe, version: 0.0.0.0, time stamp: 0x640e5ae9
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x00000000
Faulting process id: 0x0x2CD0
Faulting application start time: 0x0x1D988E2B702C89B
Faulting application path: C:\Program Files (x86)\speed-dreams-2.3.0\bin\speed-dreams-2.exe
Faulting module path: unknown
Report Id: 71200c42-18fc-4230-98f5-e03eb46e9764
Faulting package full name: 
Faulting package-relative application ID:

**Comment by stephengillie** [comment]
This 200 MB package installed normally, but errored when launched from the start menu shortcut:
> Faulting application name: speed-dreams-2.exe, version: 0.0.0.0, time stamp: 0x640e5ae9
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x00000000
Faulting process id: 0x0x1CC8
Faulting application start time: 0x0x1D988E9160D387E
Faulting application path: C:\Program Files (x86)\speed-dreams-2.3.0\bin\speed-dreams-2.exe
Faulting module path: unknown
Report Id: 999e8869-2f21-4b13-806f-cffeaba241af
Faulting package full name: 
Faulting package-relative application ID:

File: manifests/s/SpeedDreamsTeam/SpeedDreams/2.3.0/SpeedDreamsTeam.SpeedDreams.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SpeedDreamsTeam.SpeedDreams
+PackageVersion: 2.3.0
+PackageLocale: en-US
+Publisher: Speed Dreams Team
+PackageName: Speed Dreams
```
Comment (stephengillie) [inline manifests/s/SpeedDreamsTeam/SpeedDreams/2.3.0/SpeedDreamsTeam.SpeedDreams.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: http://speed-dreams.org/
PublisherSupportUrl: http://sourceforge.net/apps/trac/speed-dreams
PackageName: Speed Dreams
PackageUrl: http://speed-dreams.org/
```

**Comment by ItzLevvie** [comment]
Outside of VM it works:
image

In VM it doesn't (crashes a few seconds later):
image

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> Faulting application name: speed-dreams-2.exe, version: 0.0.0.0, time stamp: 0x640e5ae9 
> Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000 Exception code: 0xc0000005 Fault offset: 0x00000000 
> Faulting process id: 0x1d00 
> Faulting application start time: 0x01da43ed7a7f6161 
> Faulting application path: C:\Program Files (x86)\speed-dreams-2.3.0\bin\speed-dreams-2.exe 
> Faulting module path: unknown Report Id: f4ee4945-a382-49a6-a6ad-fe7f43e25060 
> Faulting package full name:  
> Faulting package-relative application ID: 

(Automated response - build 543.)

**Pull request closed without merging**